fdbserver/include/fdbserver/workloads/MetaclusterConsistency.actor.h

```diff
@@ -98,7 +98,7 @@ class MetaclusterConsistencyCheck {
 			auto allocatedItr = data.clusterAllocatedMap.find(clusterName);
 			if (!clusterMetadata.entry.hasCapacity()) {
 				ASSERT(allocatedItr == data.clusterAllocatedMap.end());
-			} else {
+			} else if (clusterMetadata.entry.clusterState == DataClusterState::READY) {
```

Comment:
Is there anything we can add an assertion for as an `else` clause for the other cluster states?

Reply:
I think I can restructure this a bit to be a more accurate